client/src/theme.js

```diff
@@ -0,0 +1,20 @@
+const theme = {
+  colors: {
+    basic: '#FFFFFF',
+    textPrimary: '#141414',
+    textSecondary: '#FFE0A6',
+    liearPrimary:
+      'linear-gradient(to bottom, #D5AE7A 0%, #917F66 50%, #575656 100%)',
+    linearSecondary:
+      'linear-gradient(to bottom, #fcf1da 0%, #d2c5aa 70%, #a79979 100%)',
+    linearTertiary:
+      'linear-gradient(to bottom, #d4ad7a 0%, #c2a074 50%, #ab906d 100%)',
+    shadow: '#00000029',
+    borderPrimary: '#AB906D',
+    borderSecondary: '#E5DDD2',
+    area: '#F2F2F2',
+    attention: 'red',
+    available: 'green',
+  },
+};
```

Comment:
Just an idea of how theming could work as well, also there was a small typo in `linearPrimary` 
```suggestion
const theme = {
  colors: {
    basic: '#FFFFFF',
    textPrimary: '#141414',
    textSecondary: '#FFE0A6',
    shadow: '#00000029',
    borderPrimary: '#AB906D',
    borderSecondary: '#E5DDD2',
    area: '#F2F2F2',
    attention: 'red',
    available: 'green',
  },
  gradients: {
   linearPrimary:
      'linear-gradient(to bottom, #D5AE7A 0%, #917F66 50%, #575656 100%)',
   linearSecondary:
      'linear-gradient(to bottom, #fcf1da 0%, #d2c5aa 70%, #a79979 100%)',
   linearTertiary:
      'linear-gradient(to bottom, #d4ad7a 0%, #c2a074 50%, #ab906d 100%)'
   }
};
```